src/pages/iou/IOUDetailsModal.js

```diff
@@ -115,6 +115,7 @@ class IOUDetailsModal extends Component {
                                 chatReportID={Number(this.props.route.params.chatReportID)}
                                 iouReportID={Number(this.props.route.params.iouReportID)}
                                 shouldHidePayButton
+                                containerStyles={[styles.cursorDefault]}
```

Comment:
Instead of resetting the pointer here. Pass the cursor style on the instance used for message preview.

Reply:
Updated and pushed new commit.